src/main/java/Parser.java

```diff
@@ -23,6 +23,9 @@ public static void handleInput(String inputPrefix, String inputSuffix, TaskList
             case "delete":
                 taskList.deleteTask(Integer.parseInt(inputSuffix) - 1);
                 break;
```

Comment:
Instead of using the the original classes which handles the making of new classes associated with keywords, you can have separate command classes so that these classes would only handle one task.